### PR TITLE
doc: Add 23.0 release notes

### PR DESCRIPTION
**Pull request by laanwj**
Add 23.0 release notes from the wiki. We forgot to move these back to the branch, so they go direct-to-master.

Only change from the wiki is to add the credits.

### CODE REVIEW

**Comment by dunxen** [comment]
ACK bf4594f

**Comment by Kvaciral** [comment]
```
- Martin Ankerl
- Martin Leitner-Ankerl
```
double.. and
```
- merge-script
```
is obviously not a nick, but a name of a script or bot which have no place in this list..

File: doc/release-notes/release-notes-23.0.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+- Heebs
+- Hennadii Stepanov
+- hg333
+- HiLivin
+- Igor Cota
+- Jadi
+- James O'Beirne
+- Jameson Lopp
+- Jarol Rodriguez
+- jarolrod
```
Comment (laanwj) [inline doc/release-notes/release-notes-23.0.md:290]: Oh no another one

Comment (mzumsande) [reply doc/release-notes/release-notes-23.0.md:290]: Some more I found:
Bruno Garcia / brunoerg
Jon Atack / jonatack
Russell Yanofsky / Ryan Ofsky

**Comment by Kvaciral** [comment]
```
- Jon Atack
- jonatack
```
:P

**Comment by laanwj** [comment]
I guess we could avoid some of those duplicates by automatically deduplicating based on mail address and/or combine with a mailmap in the release process (https://github.com/bitcoin/bitcoin/blob/master/doc/release-process.md#write-the-release-notes). Anyhow, normally these are found early during review in the wiki, not last minute.

File: doc/release-notes/release-notes-23.0.md (hunk 1 above)
Comment (laanwj) [reply doc/release-notes/release-notes-23.0.md]: Thanks!

> Russell Yanofsky / Ryan Ofsky

These are actually the same person, right?

**Comment by dunxen** [comment]
Re-ACK  2abfb6c 

> ```
> - merge-script
> ```
> 
> is obviously not someone's nick, but a name of a script or bot which have no place in this list..

Poor `merge-script` :(

**Pull request merged**